### ISSUE TITLE
Planets with colors

### ISSUE THREAD

**Issue by ThiagoLopes** [mochaoss/inspace#14]
Add "unique" color for every planet to show in tags.
[Exemple](http://vimawesome.com/)

**Comment by ThiagoLopes** [comment]
What I thought the way it would be:
* A list with some code reference for a color in bulma colors palette.
* Every time when a planet is created * sort * with one code
* Check: if last planet created have same code, re-sort. if not associate this code to planet

**Comment by cacarrara** [comment]
Check not only last planet, but if any planet has the same color code picked for the current planet.

**Comment by LucasBSousa** [comment]
@ThiagoLopes your propposal leaves room for the colour codes to give some very ugly colros that don't match and would simply just look bad. **Keep in mind there are way more ugly colors than good looking colors**. Maybe we could decide on a handful of colors (i know it might end up being a lot of them), create css classes for each color (color1, color2, color3, so on...), and when a new planet is created, it gets associated with the lowest class number available. We should store the value in the database, so that we can use `class{colorNumber}` or something like that.

**Comment by ThiagoLopes** [comment]
@LucasBSousa 
Yes, we can use this [table](http://bulma.io/documentation/overview/colors/).
this table have the colors used in bulma framework
